packages/vitest/src/utils/base.ts

```diff
@@ -112,3 +112,11 @@ export function assertTypes(value: unknown, name: string, types: string[]): void
   const pass = types.includes(receivedType)
   if (!pass) throw new TypeError(`${name} value must be ${types.join(' or ')}, received "${receivedType}"`)
 }
+
+/**
+ * joinArrayItems
+ * convert Array<T> to a single string to display in stdout
+ */
+export function joinArrayItems(items: unknown[]): string {
```

Comment:
just out of interest, what is the difference between just calling `.join`? I see all test cases will return the same if called with `.join`

Reply:
Fixed to use `join` since `include` and `watchIgnore`  are absolutely the array. Thanks.